regulations/views/section.py

```diff
@@ -7,10 +7,10 @@
 from regulations.generator.toc import fetch_toc
 from regulations.generator.section_url import SectionUrl
 from regulations.views import error_handling
-from regulations.views.mixins import SidebarContextMixin, CitationContextMixin
+from regulations.views.mixins import SidebarContextMixin
 
 
-class SectionView(SidebarContextMixin, CitationContextMixin, TemplateView):
+class SectionView(SidebarContextMixin, TemplateView):
```

Comment:
Since this is shown to model something more abstract it can be renamed to something like `ReaderView`